pyomo/opt/tests/solver/test_shellcmd.py

```diff
@@ -281,8 +292,13 @@ def test_executable_isexe_abspath_user(self):
             with SolverFactory(name, executable=isexe_abspath_user) as opt:
                 if isinstance(opt, UnknownSolver):
                     continue
-                self.assertEqual(opt._user_executable, isexe_abspath)
-                self.assertEqual(opt.executable(), isexe_abspath)
+                # Note the user path could be different from our
```

Comment:
This isn't related, correct? It's just an extra test that is nice to have?

Reply:
Worse: those assertions are not valid and will fail if we are running in a directory (like a user's HOME directory) that is accessed through symbolic links.